ert_storage/endpoints/records.py

```diff
@@ -227,7 +224,10 @@ async def post_ensemble_record_matrix(
         )
         content_type = "text/csv"
 
-    if prior_id is not None and record_class != "parameter":
+    ensemble = _get_and_assert_ensemble(db, ensemble_id, name, realization_index)
+    is_parameter = name in ensemble.parameter_names
+    is_response = name in ensemble.response_names
+    if prior_id is not None and not is_parameter:
```

Comment:
not related to this code-review but more a question in general. My PyCharm insists on using i.e `prior_id is not None` as you do at this point, but we are not consistent on this. Some of us using `==` / `!=` and some `is` / `is not`. I think i'm using both 😅  . Should we align and do the same all over?

Reply:
I have been consistently using `is not None` and `==` for `True`, `False` since I started writing Python. `is` should be faster due to it being an identity check, so CPython has to do less work, plus it's readable.

Reply:
Ok, then you are consistent on this. I'm not sure I am - but i want to be 😅  I guess i will start to make some comments on this when discovering inconsistency - which i see quite often.

Reply:
My peeve is `dict` instead of dict literals. Both get used in the same function call and it's unsettling to say the least. Dict literals (`{}`) are better in every way except having to use quotes around the keys. 

1. They're more flexible (`{**foo, "foo": "bar", **bar}` to combine dicts
2. They're more performant https://stackoverflow.com/a/6612024